reporting/html/report.tmpl

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>2MS Scan Result</title>
+</head>
+<style>
+{{ includeCSS }}
+</style>
+<body>
+  {{ includeLogo }}
+  <div class="container">
+      <div class="report-header-footer">
+        <span class="title">
+          2MS
+          <span>REPORT</span>
+        </span>
+      </div>
+
+  <div class="run-info">
+    <span style="flex-basis:100%" id="scan-paths"><strong>Total items scanned:</strong> {{ .TotalItemsScanned }}</span>
+    <span style="flex-basis:100%" id="scan-paths"><strong>Total serets found:</strong> {{ .TotalSecretsFound }}</span>
+  </div>
+  <hr class="separator">
+
+  {{ range $file, $secrets := .Results }}
+
+  <div class="secret-info">
+  <h3>{{ $file }}</h3>
+
+  <ul>
+    {{ range $index, $secret := $secrets }}
+
+      <div class="secret-info-details">
+        <span><strong>ID:</strong> {{ .ID }}</span>
+        <span><strong>Source:</strong> {{ .Source }}</span>
```

Comment:
Most of the times, the `Source` will be a link, but not for Git, for example.

Do you think we can make it a link when it is?

Reply:
looking into it